src/taipy/templates/taipy-default-template/hooks/post_gen_project.py

```diff
@@ -10,8 +10,178 @@
 
+
+def handle_services(use_rest, use_core):
+    if use_core in ["YES", "Y"] or use_rest in ["YES", "Y"]:
+        # Write "import taipy as tp" at the third line of the import.txt file
+        with open(os.path.join(os.getcwd(), "sections", "import.txt"), "r") as import_file:
+            import_lines = import_file.readlines()
+            import_lines[1] += "import taipy as tp\n"
+        with open(os.path.join(os.getcwd(), "sections", "import.txt"), "w") as import_file:
+            import_file.writelines(import_lines)
+
+    if use_rest in ["YES", "Y"]:
+        with open(os.path.join(os.getcwd(), "sections", "import.txt"), "a") as import_file:
+            import_file.write("from taipy import Rest\n")
+        with open(os.path.join(os.getcwd(), "sections", "main.txt"), "a") as main_file:
+            main_file.write("    rest = Rest()\n")
+
+    if use_core in ["YES", "Y"]:
+        # Import and run Core service if Rest service is not run yet
+        if use_rest not in ["YES", "Y"]:
+            with open(os.path.join(os.getcwd(), "sections", "import.txt"), "a") as import_file:
+                import_file.write("from taipy import Core\n")
+            with open(os.path.join(os.getcwd(), "sections", "main.txt"), "a") as main_file:
+                main_file.write("    core = Core()\n")
+
+        # Create and submit the placeholder scenario
+        with open(os.path.join(os.getcwd(), "sections", "main.txt"), "a") as main_file:
+            main_file.write("    # #############################################################################\n")
+            main_file.write("    # PLACEHOLDER: Initialize your data application here                          #\n")
+            main_file.write("    #                                                                             #\n")
+            main_file.write("    # Example:                                                                    #\n")
+            main_file.write("    # from configuration import scenario_config                                   #\n")
+            main_file.write("    # scenario = tp.create_scenario(scenario_config)                              #\n")
+            main_file.write("    # scenario.submit()                                                           #\n")
+            main_file.write("    # Comment, remove or replace the previous lines with your own use case        #\n")
+            main_file.write("    # #############################################################################\n")
+    else:
+        shutil.rmtree(os.path.join(os.getcwd(), "algorithms"))
+        shutil.rmtree(os.path.join(os.getcwd(), "configuration"))
+
+
+def handle_run_service():
+    with open(os.path.join(os.getcwd(), "sections", "main.txt"), "a+") as main_file:
+        main_file.seek(0)
+        main_content = main_file.read()
+
+        if "rest = Rest()" in main_content:
+            main_file.write('    tp.run(gui, rest, title="{{cookiecutter.__application_title}}")\n')
```

Comment:
We cannot create scenario before having run Core. The run of the Core should be before the placeholder to create/submit scenarios...